custom_components/tahoma/switch.py

```diff
@@ -112,3 +127,46 @@ async def async_toggle(self, **_):
     def is_on(self):
         """Get whether the switch is in on state."""
         return self.select_state(CORE_ON_OFF_STATE, IO_FORCE_HEATING_STATE) == STATE_ON
+
+
+class TahomaLowSpeedSwitch(TahomaEntity, SwitchEntity, RestoreEntity):
+    """Representation of Low Speed Switch."""
+
+    _attr_icon = "mdi:feather"
+
+    def __init__(self, device_url: str, coordinator: TahomaDataUpdateCoordinator):
+        """Initialize the low speed switch."""
+        super().__init__(device_url, coordinator)
+        self._is_on = False
+
+    async def async_added_to_hass(self):
+        """Run when entity about to be added."""
+        await super().async_added_to_hass()
+        state = await self.async_get_last_state()
+        if state:
+            self._is_on = state.state == STATE_ON
```

Comment:
Looks like a good fix, but is there no helper to restore the state? I would say that this is something that RestoreEntity should have already. Or do they only offer `async_get_last_state`?

Reply:
https://github.com/home-assistant/core/blob/dev/homeassistant/helpers/restore_state.py#L241
That was my first though too. But after a discussion on Discord, and some research within the Core, it seems the correct pattern.

Reply:
For instance: https://github.com/home-assistant/core/blob/a59460a23336627d0bc12b1eefffdaa516e55e87/homeassistant/components/switchbot/switch.py#L46

Reply:
Weird, but looks ok :-)